### PR TITLE
[fix] change timeout interval

### PR DESCRIPTION
**Pull request by steins-code**
increase timeout interval to avoid wrong internet off states

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift [hunk 1]
```diff
@@ -186,6 +187,8 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 	}
 
 	private func presentHomeVC() {
+		Connectivity.urlSessionConfiguration.timeoutIntervalForRequest = 15.0
+		Connectivity.urlSessionConfiguration.timeoutIntervalForResource = 15.0
```
Comment (haosap) [inline src/xcode/ENA/ENA/Source/SceneDelegate.swift:191]: what are the old values?

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift [hunk 2]
```diff
@@ -186,6 +187,8 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
...
 		enStateHandler = ENStateHandler(
 			initialExposureManagerState: exposureManager.preconditions(),
 			reachabilityService: ConnectivityReachabilityService(
```
Comment (haosap) [inline src/xcode/ENA/ENA/Source/SceneDelegate.swift:194]: In fact, the EnStateHandler has a strong reference to ConnectivityReachabilityService and meanwhile, ConnectivityReachabilityService also keeps the strong ref to EnStateHandler. please check it.

**Review by haosap — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift (hunk 1 above)
Comment (ChristianKienle) [reply src/xcode/ENA/ENA/Source/SceneDelegate.swift:191]: 5s

**Review by haosap — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Source/SceneDelegate.swift:191]: 5s

**Review by ChristianKienle — APPROVED** [review]

**Pull request merged**